docs/docs/general/02-Getting Started/writing-your-first-test.md

```diff
@@ -4,9 +4,22 @@ sidebar_position: 3
 
 # Writing Your First Test
 
+:::tip
+What you are going to learn
+
+- Initialize your first automation suite
+- Configure your credentials (workspace id and access token)
+- Understand the generated automation case
+- Run your first automation
+- Troubleshooting any issues
+- Where to go next
+:::
+
 ## Quickstart
```

Comment:
Maybe we delete the `Quickstart` header. I think we don't need it anymore after your additions